### DISCUSSION TITLE
Make jetclient request global over different repo's

### DISCUSSION THREAD

**Discussion by tiboleemans** [JetClient/jet-client-support#40] [Q&A] [answered]
Is it possible to make the JetClient collection global across different repositories, like the databases in IntelliJ.

I suppose these collections are scoped by repo because they can be managed by the git repo. But it would be very usefull to use them across IntelliJ globally.

**Comment by AntonShuvaev** [comment, marked as answer]
Thank you for the suggestion! This is a good idea, and I'll consider adding it in future releases.

As a workaround, you can manually copy the .idea/JetClient folder from one project and paste it into another to transfer the request collection from one project to another.
